### DISCUSSION TITLE
Test simulating 100K(s) of users

### DISCUSSION THREAD

**Discussion by epitka** [VIP-Logic/NBomber#560] [General]
Would NBomber be suitable tool to test executing several hundred K of browsers using Playwritght scripts. We have spikes when something goes on sale that we sometimes see 400-500 K users trying to buy items, and we would like to write a test to simulate it. Wondering if NBomber could be used for this?

**Comment by AntyaDev** [comment]
Hi @epitka ,
I never tried to simulate such a scenario by wrapping Playwright into NBomber, and I can't give you figures.
NBomber is using multithreading very effectively, and it provides an NBomber Cluster that can be used to run millions of users.

Do you know how many Playwright scripts can you run under a single NBomber process?
